cmd/node/node.go

```diff
@@ -360,6 +363,45 @@ func (app SpacemeshApp) stopServices() {
 
 }
 
+func isFileExist(name string) (bool, error) {
+	_, err := os.Stat(name)
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return err != nil, err
+}
+
+func getEdIdentity() (*signing.EdSigner, error) {
+	dataDir, err := filesystem.GetSpacemeshDataDirectoryPath()
+	if err != nil {
+		log.Error("Could not get data path err=%v", err)
+		return signing.NewEdSigner(), err
+	}
+
+	f := dataDir + "/identity.ed"
+	if exist, _ := isFileExist(f); exist {
```

Comment:
You don't need this part, you can tell that the file doesn't exist from the error returned when calling `ReadFile`

Reply:
Fixed